cast/src/main.rs

```diff
@@ -53,8 +53,11 @@ enum Commands {
 #[tokio::main]
 async fn main() -> Result<()> {
     let cli = Cli::parse();
+    // TODO #2147
+    if !cli.accounts_file_path.exists() {
+        anyhow::bail!("accounts file ({}) does not exist", cli.accounts_file_path);
+    }
 
-    // todo: #2052 take network from scarb config if flag not provided
```

Comment:
can you point me to where it's done? sorry, I can't find it

Reply:
[fdbck](https://github.com/software-mansion/protostar/pull/2138/commits/c18edf2e55891fc1040b8fa3fc3614efa658116b)